src/main/java/Parser.java

```diff
@@ -0,0 +1,71 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.InputMismatchException;
+
+import static java.lang.Integer.parseInt;
+
+public class Parser {
```

Comment:
Should it be the responsibility of the parser to parse all string input into objects before passing it to the respective commands?